prompts/next-intl-add-language.prompt.md

```diff
@@ -0,0 +1,19 @@
+---
+mode: "agent"
+description: "Add new language to a NextJS with next-intl application"
+---
+
+This is a guide to add a new language to a Next.js project using next-intl for internationalization, following steps from https://next-intl.dev/docs/getting-started/app-router/with-i18n-routing.
```

Comment:
The agent won't follow an external link with `#fetch` being used, so that would need to be a tool.